### DISCUSSION TITLE
opencascade.js导入step文件并解析

### DISCUSSION THREAD

**Discussion by qingmeng112** [donalffons/opencascade.js#258] [Q&A]
我想通过官网给的react的例子，自己增加了一个导入一个外部step文件的按钮，想通过opencascade.js实现解析并渲染的功能，但是我创建的STEPCAFControl_Reader_1()对象使用ReadFile()方法去读取我项目中public目录下的step文件，在通过PrintCheckLoad_1()方法去检查是否解析成功时，控制台输出了无效数据的错误，在通过model-vivewer渲染的时候也出现了未定义的材质属性错误

**Comment by frank-pian** [comment]
把完整代码和报错贴出来，你这样描述猜不出什么原因。
